Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/MRTKUGUIInputField.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// A derived class of UGUI's InputField to workaround with some issues of typing on HoloLens 2
+    /// </summary>
+    public class MRTKUGUIInputField : InputField
+    {
+#if UNITY_2019_3_OR_NEWER
+        [SerializeField]
+        private bool disableUGUIWorkaround = false;
+
+        /// <summary>
+        /// Currently there is a Unity bug that needs a workaround. Please keep this setting to be false until an announcement of the version of Unity/UGUI that resolves the issue is made.
+        /// </summary>
+        public bool DisableUGUIWorkaround
```

Comment:
Could this be internal? Or do we expect devs to set this at runtime?

Reply:
I am not sure so I just left it as public to be safe.